src/uiSelectController.js

```diff
@@ -112,6 +112,10 @@ uis.controller('uiSelectCtrl',
 
       $scope.$broadcast('uis:activate');
 
+      $timeout(function(){
```

Comment:
Honestly I don't remember, but I guess it was added to be sure it's triggered after the main cycle.